packages/angular-query-experimental/src/types.ts

```diff
@@ -137,7 +137,7 @@ export interface CreateMutationOptions<
   TContext = unknown,
 > extends Omit<
     MutationObserverOptions<TData, TError, TVariables, TContext>,
-    '_defaulted' | 'variables'
```

Comment:
There is no 'variable' field to omit in MutationObserverOptions

Reply:
All adapters except Vue Query contain this type. So I think it should be changed for all of those?

Reply:
Sure, thanks for your eagle eyes 👍